### DISCUSSION TITLE
Is it possible to have Mapsui running in a Console application to export map for automation purposes

### DISCUSSION THREAD

**Discussion by stobon17** [Mapsui/Mapsui#2215] [Q&A] [answered]
Can use Mapsui in a console application, build a map with multiple layers (no UI) and export the map as a PNG. I know how to build the map and export it in a WPF application. Just curious if it is possible to run in a console app to automate this process without the need for a client present in either a web page or WPF application.

**Comment by inforithmics** [comment, marked as answer]
I would look at
Tests/Mapsui.Rendering.Skia.Tests
Because there are Images created off all Samples as a Regression Test

**Reply by stobon17** [reply]
just what i needed, thank you!
